pkgs/tools/networking/offlineimap/default.nix

```diff
@@ -23,7 +23,10 @@ python2Packages.buildPythonApplication rec {
   doCheck = false;
 
   nativeBuildInputs = [ asciidoc libxml2 libxslt docbook_xsl ];
-  propagatedBuildInputs = with python2Packages; [ six kerberos rfc6555 ];
+  propagatedBuildInputs = with python2Packages; [ six kerberos rfc6555 ]
+                                             ++ (if enableProxy
```

Comment:
FYI, it would be better to use `lib.optional` instead of `if` (but I think pysocks should be unconditionally added).